package.json

```diff
@@ -31,10 +31,11 @@
     "re:clean": "rescript clean",
     "re:start": "rescript build -w",
     "start:dev": "npm run re:start & npm run start",
-    "start:playground": "cd Hyperswitch-React-Demo-App && node promptScript.js && npm run start",
+    "start:playground": "npm run postinstall && cd Hyperswitch-React-Demo-App && node promptScript.js && npm run start",
```

Comment:
Please sure that these cmds runs sequentially .. at times i have seen prompts being thrown before the previous cmd executes (in this case npm i)